<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Water Maper (Database Section)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::tab:disabled { width: 0; height: 0; margin: 0; padding: 0; border: none; }

QPushButton{
 background-color: rgba(116, 121, 115, 95);

	color: rgb(255, 255, 255);
font: 11pt &quot;Arial&quot;;
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}

QPushButton:pressed {
background-color: rgba(116, 121, 115, 235);
color:rgba(255, 248, 230,255);
}

QPushButton:hover {
background-color: rgba(238, 234, 209, 125);
font: 12pt &quot;Arial&quot;;
}
QLabel{
background-color: rgba(211, 236, 231, 85);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}
QTextEdit{
background-color: rgba(241, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}
QDateEdit{
background-color: rgba(251, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}
QTableView{
background-color: rgba(108, 98, 88, 100);
color: rgb(255, 255, 255);
border-radius:10px;
}
QWebView{
background-color: rgba(241, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:10px;
}
QLineEdit{
background-color: rgba(241, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}
QComboBox{
background-color: rgba(251, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}

QGroupBox{

border-width: 1px;
border-style: solid;
border-color: grey;
background-color: rgba(58, 54, 51, 70);
font: 11pt &quot;Arial&quot;;

color: rgb(255, 255, 255);
}
QTabWidget QWidget{
font: 10pt &quot;Bitstream Vera Sans&quot;;
color: rgb(255, 255, 255);
}

QTabBar::tab:hover {
border-top:1px solid rgb(111, 111, 111);
border-bottom:5px solid rgb(111, 111, 111);
font: 14pt &quot;Bitstream Vera Sans&quot;;
}
QTabBar::tab {
border-top:2px solid rgba(255, 248, 230,190);
font: 12pt &quot;Bitstream Vera Sans&quot;;
height: 25px; width: 150px;
}
QTabBar::tab:selected {
  border-top:4px solid rgba(166, 166, 166,90);
}


QTabWidget::pane
{
border-width: 1px;
border-style: solid;
border-color: grey;
background-color: rgba(58, 54, 51, 70);
}</string>
  </property>
  <widget class="QPushButton" name="btn_upload">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbx_tech">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>/var/www/html/test.txt</string>
   </property>
  </widget>
  <widget class="QTableView" name="tv_csv_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_server">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>121</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Server Setup</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="tbx_port">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>3306</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>Pass:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_db_pass">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>root</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_db_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>water_map</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>Host:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_host">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>localhost</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>Database Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_db_user">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>root</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Setup</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hide">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>tbx_host</zorder>
   <zorder>label_8</zorder>
   <zorder>label_4</zorder>
   <zorder>btn_setup</zorder>
   <zorder>label_7</zorder>
   <zorder>tbx_db_name</zorder>
   <zorder>tbx_db_user</zorder>
   <zorder>tbx_port</zorder>
   <zorder>tbx_db_pass</zorder>
   <zorder>btn_hide</zorder>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>710</y>
     <width>1331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>Upload Your (.csv) File,</string>
   </property>
  </widget>
  <widget class="QTableView" name="tv_output">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>851</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>Showing the Database Table (Output Results) :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_test">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exp_csv">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>470</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Export to (.csv)</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_uload">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Map">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>MAP</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWebView" name="web_View">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>851</width>
     <height>451</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Database table Setup:</string>
   </property>
   <widget class="QPushButton" name="btn_marge">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>Marge With Existing Schema and Table</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>TABLE:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_table_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>user_input</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_schema">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>water_map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_insert">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>Insert Into Existing Schema and Table</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>SCHEMA:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="text">
     <string>Create with a New Schema</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>Showing the CSV File :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbx_search_csv">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>Search:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>470</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>Search:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbx_search_dbs">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>470</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>btn_upload</zorder>
  <zorder>grp_server</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_12</zorder>
  <zorder>btn_test</zorder>
  <zorder>btn_exp_csv</zorder>
  <zorder>btn_uload</zorder>
  <zorder>btn_Map</zorder>
  <zorder>web_View</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_11</zorder>
  <zorder>tbx_tech</zorder>
  <zorder>label_13</zorder>
  <zorder>tbx_search_csv</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>tbx_search_dbs</zorder>
  <zorder>tv_output</zorder>
  <zorder>tv_csv_table</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
